src/LinearMap.jl

```diff
@@ -124,3 +124,38 @@ that ``σ(d, L) = M⋅σ(M^T d, S)`` for any direction ``d``.
 function σ(d::AbstractVector{<:Real}, lm::LinearMap)::AbstractVector{<:Real}
     return lm.M * σ(lm.M.' * d, lm.sf)
 end
+
+"""
+    ∈(x::AbstractVector{N}, lm::LinearMap{<:LazySet, N})::Bool where {N<:Real}
+
+Check whether a given point is contained in a linear map of a convex set.
+
+### Input
+
+- `x`  -- point/vector
+- `lm` -- linear map of a convex set
+
+### Output
+
+`true` iff ``x ∈ lm``.
+
+### Algorithm
+
+This implementation inverts the matrix: ``x ∈ M⋅S`` iff ``M^{-1}⋅x ∈ S``.
+
+### Examples
+
+```jldoctest
+julia> lm = LinearMap([2.0 0.0; 0.0 1.0], BallInf([1., 1.], 1.));
+
+julia> ∈([5.0, 1.0], lm)
+false
+julia> ∈([3.0, 1.0], lm)
+true
+```
+"""
+function ∈(x::AbstractVector{N},
+           lm::LinearMap{<:LazySet, N})::Bool where {N<:Real}
+    @assert length(x) == dim(lm)
+    return ∈(inv(lm.M) * x, lm.sf)
```

Comment:
what happens if `M` is not invertible?

it's preferable to divide via `lm.M \ x` instead of computing the full matrix inverse explicitly with `inv`, which is not needed.

Reply:
Do you mean `return ∈(lm.M \ x, lm.sf)`?

Reply:
yes. this covers the invertible case, but doesn't answer the problem generally i think. let

```julia
julia> B = BallInf(zeros(4), 1.);
julia> M = [1. 0 0 0; 0 1 0 0]/2;
julia> x = ones(4.);
```

then the point `[0.5, 0.5]` is in the image set of the linear map of `B` under `M`. however the check of above will fail in `M \ x`.

Reply:
on a side note:
```julia
julia> dim(M*B)
2
```

Reply:
if you want we can say in the docstring that this method applicable to the invertible case for now.